gwtquery-core/src/main/java/com/google/gwt/query/client/plugins/effects/PropertiesAnimation.java

```diff
@@ -70,61 +70,61 @@
    *
    */
   public static enum EasingCurve implements Easing {
-    linear(0, 0, 1, 1) {
+    LINEAR(0, 0, 1, 1) {
```

Comment:
Changing this will break the public API of GQuery these names are aligned with jquery easing names.